pages/index.js

```diff
@@ -43,6 +45,25 @@ const Home = () => {
               <div className="absolute-pos">
                 <h1>Ouro Payments</h1>
                 <h4>Privacy preserving continuous payment streams</h4>
+                <h4>
+                  Use Ouro for
+                  <TypistLoop interval={1000}>
+                    {[
+                      "Salary payments",
+                      "Car rental payments",
+                      "Service subscriptions",
```

Comment:
I am not sure what are the best examples to include. I've given it a bunch of thought, but still not sure on this copy yet.